management/grant.go

```diff
@@ -32,3 +32,9 @@ func (m *GrantManager) List(opts ...ListOption) (g []*Grant, err error) {
 	err = m.get(m.uri("grants")+m.q(opts), &g)
 	return
 }
+
+// Delete revokes a grant associated with a user-id
+// https://auth0.com/docs/api/management/v2#!/Grants/delete_grants_by_id
+func (m *GrantManager) Delete(userID string, opts ...ListOption) error {
```

Comment:
From the documentation it seems that the argument here should be id, not userID. A user ID may be supplied using a parameter.

Example:

https://alexkappa.eu.auth0.com/api/v2/grants/123
https://alexkappa.eu.auth0.com/api/v2/grants/123?user_id=567

As per the documentation the grant id is a required field.

Reply:
fixed in [9d61ad3](https://github.com/go-auth0/auth0/pull/91/commits/9d61ad38e774c0810f1c56e5c26480b8cd457f01)